lib/crypto/test/crypto_SUITE.erl

```diff
@@ -665,6 +667,23 @@ rand_plugin_s() ->
 rand_plugin_s(Config) when is_list(Config) ->
     rand_plugin_aux(explicit_state).
 
+%%--------------------------------------------------------------------
+cipher_info() ->
+    [{doc, "crypto cipher_info testing"}].
+cipher_info(Config) when is_list(Config) ->
+    #{type := _,key_length := _,iv_length := _,
+        block_size := _,mode := _} = crypto:cipher_info(aes_128_cbc),
+    {'EXIT',_} = (catch crypto:cipher_info(not_a_cipher)),
+    ok.
+
+%%--------------------------------------------------------------------
+hash_info() ->
+    [{doc, "crypto hash_info testing"}].
+hash_info(Config) when is_list(Config) ->
+    #{type := _,size := _,block_size := _} = crypto:hash_info(sha3_512),
```

Comment:
sha3_512 is not available in older crypto_libs. Change it to for example sha256 to make the test pass on machines with such libs.

Reply:
Done.